content/docs/hooks-recipes.md

```diff
@@ -10,3 +10,157 @@ prev: hooks-faq.html
 This is a collection of recipes that may help you start "thinking in Hooks". Feel free to play with them or copy and paste them into your project or tweak them to your needs.
 
 ## Data Fetching
+
+## Subscriptions
+
+Sometimes React components need to subscribe to an external data source. The `useSubscription` hook below shows how to do this in a way that is compatibile with React's upcoming concurrent rendering mode.
```

Comment:
I would maybe rephrase it so that it doesn't feel like CM is the *problem* here.

Maybe:

> Sometimes React components need to subscribe to an external data source. Handling all the possible edges in a performant way can be challenging, which is why people often use dedicated third-party libraries for connecting to data. The `useSubscription` Hook below simplifies that. It lets you subscribe to an external data source while handling all the edge cases. It is also compatible with the upcoming concurrent rendering mode.